src/styles/components/_detailsBanner.scss

```diff
@@ -1,18 +1,30 @@
+$breakpoint-tablet: 820px;
+
 .details-banner {
   font-weight: 700;
-  font-size: font-size("sm");
-  line-height: 4.6667rem; // 28px
+  line-height: 2.1rem; // 28px
   letter-spacing: 2%;
-  margin: 1.25rem;
+
+  h1 {
+    font-size: 1.5625rem;
+  }
 }
 
 .details-banner-image {
   box-sizing: border-box;
   background-color: white;
-  padding: 1.25rem; // 20px
+  border: 3px solid black;
```

Comment:
Just a quick note that this border looks like it's 2px in Figma

Reply:
Thanks, totally missed that! Fixed it in 522bf2c.